x-pack/plugins/ml/public/application/services/elastic_models_service.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type ModelDefinitionResponse } from '@kbn/ml-trained-models-utils';
+import { type TrainedModelsApiService } from './ml_api_service/trained_models';
+
+export type ElserVersion = 1 | 2;
```

Comment:
as this is for external consumption, this type should probably also live in the `'@kbn/ml-trained-models-utils'` package, just in case other people need it.

Reply:
https://github.com/elastic/kibana/pull/167000/commits/d97cf1db330af7814a4d8a3895e1b3c918ba63fa